sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -0,0 +1,374 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (c) 2019 Intel Corporation
+
+/*
+ *  sdw_rt711_rt1308_rt715 - ASOC Machine driver for Intel SoundWire platforms
+ * connected to 3 Realtek devices
+ */
+
+#include <linux/acpi.h>
+#include <linux/async.h>
+#include <linux/delay.h>
+#include <linux/device.h>
+#include <linux/gpio.h>
+#include <linux/init.h>
+#include <linux/input.h>
+#include <linux/io.h>
+#include <linux/dmi.h>
+#include <linux/module.h>
+#include <linux/slab.h>
+#include <sound/jack.h>
+#include <sound/pcm.h>
+#include <sound/pcm_params.h>
+#include <sound/soc.h>
+#include <sound/soc-acpi.h>
+#include "../../codecs/hdac_hdmi.h"
+
+struct mc_private {
+	struct list_head hdmi_pcm_list;
```

Comment:
shouldnt this be inside #if IS_ENABLED(CONFIG_SND_SOC_HDAC_HDMI) too?

Reply:
doesn't hurt this way, this is how we've handled HDMI on all other machine drivers.